web/src/components/tooltips/TooltipWrapper.tsx

```diff
@@ -1,15 +1,15 @@
 import * as Tooltip from '@radix-ui/react-tooltip';
```

Comment:
This component now supports both:
- Hover tooltips like the ones on the sidebar
- Clickable tooltips (exchange arrows) and making sure only one of them is open at a time